pkg/platform/factory/factory.go

```diff
@@ -32,12 +32,14 @@ func CreatePlatform(parentLogger logger.Logger,
 	platformType string,
 	platformConfiguration interface{}) (platform.Platform, error) {
 
+	containerBuilderConfiguration := kube.GetContainerBuilderConfiguration(platformConfiguration)
```

Comment:
1. move to 'kube' case
2. move `GetContainerBuilderConfiguration` to a private method of factory